net/phetch/Portfile

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo 1.0
+PortGroup           github 1.0
+
+github.setup        xvxx phetch 1.1.0 v
+
```

Comment:
Please add a `revision` field

Reply:
OK. I have added the line to the Portfile. If there is anything else, please let me know. Thank you.